stl/inc/utility

```diff
@@ -644,6 +645,77 @@ template <size_t _Idx>
 inline constexpr in_place_index_t<_Idx> in_place_index{};
 #endif // _HAS_CXX17
 
+#if _HAS_CXX20
+
+template <typename _Ty>
+inline constexpr bool _Is_integral_not_bool_char = is_integral_v<_Ty>

```

Comment:
I think it would be better to make this a concept
```suggestion
concept _Integral_not_bool_char = is_integral_v<_Ty>
```

Reply:
Yes, I forgot about the concepts. I still used C++17.

Reply:
I dont know about the plans for moving the requirement up to clang-10 which would benecessary to support concepts.

That said if you do go with concepts you should use them in the functions so that the function is SFINAED out.
Otherwise it should be `enable_if_t`